File: Lab07/app/src/main/java/com/example/lab07/MainActivity.java
package com.example.lab07;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentTransaction;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        enableEdgeToEdge();
        setContentView(R.layout.activity_main);

        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, RegisterFragment.newInstance())
                    .commitNow();
        }
    }

    private void enableEdgeToEdge() {
        // Implementación de edge-to-edge según tus necesidades
        // Por ejemplo, para habilitar el borde a borde en la ventana:
        // getWindow().getInsetsController().show(WindowInsets.Type.statusBars() | WindowInsets.Type.navigationBars());
    }
}

File: HangmanGame/app/src/main/java/com/example/hangmangame/MainActivity.kt
package com.example.hangmangame

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    private lateinit var canvasView: CanvasView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        canvasView = findViewById(R.id.canvasView)

        findViewById<Button>(R.id.btnPlay).setOnClickListener {
            canvasView.nextPart()
        }

        findViewById<Button>(R.id.btnReset).setOnClickListener {
            canvasView.reset()
        }
    }
}
